### DISCUSSION TITLE
Can ES module share __webpack_require__ runtime with normal js file?

### DISCUSSION THREAD

**Discussion by szqsy17** [webpack/webpack#18825] [Q&A]
Can ES module share `__webpack_require__` runtime with normal js file?

**Comment by alexander-akait** [comment]
You can, but I don't recommend, it is a internal api and can changed

**Reply by szqsy17** [reply]
How can I do that？

**Comment by szqsy17** [comment]
Can use processAssets hook to export `__webpack_require__` in runtime.js

**Reply by alexander-akait** [reply]
Yes, you can
